packages/pretty-format/src/index.js

```diff
@@ -405,13 +456,13 @@ function prettyFormat(val, opts) {
     indent = createIndent(opts.indent);
     refs = [];
     const pluginsResult = printPlugin(val, indent, prevIndent, spacing, edgeSpacing, refs, opts.maxDepth, currentDepth, opts.plugins, opts.min, opts.callToJSON, opts.printFunctionName, opts.escapeRegex, colors);
-    if (pluginsResult) {
+    if (pluginsResult !== null) {
       return pluginsResult;
     }
   }
 
   const basicResult = printBasicValue(val, opts.printFunctionName, opts.escapeRegex);
-  if (basicResult) {
+  if (basicResult !== null) {
```

Comment:
Should these checks really be `!== null`? Technically you are changing behavior here from what it was before. Should we do `!= null` to also catch undefined here?